sympy/physics/mechanics/abstract_body.py

```diff
@@ -0,0 +1,136 @@
+from abc import ABC, abstractmethod
+
+from sympy.core.backend import Symbol, sympify
+from sympy.physics.vector import Point
+
+__all__ = ['_Body']
+
+
+class _Body(ABC):
```

Comment:
Are you happy that this is the name we'd like to go for? I've worked on other projects where the convention has been to call this `_Body` and others where it has been `BodyABC`. Is there anywhere else in SymPy where ABCs are used and we can make sure we're consistent? I will also make sure that I'm consistent with what we decide here for the ABCs in `sympy.physics.biomechanics`.

Reply:
The abstract method for `KanesMethod` and `LagrangeMethod` is also called `_Method`, which is in the file `method.py`

Reply:
If we're going with `BodyBase` then it would make sense for `_Method` to become `MethodBase`. I assume this isn't problematic because the leading underscore indicates that it's private anyway. But something for a different PR.